source/features/pr-earliest-tag-after-merge.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+import oneTime from 'onetime';
+import TagIcon from 'octicon/tag.svg';
+import {observe} from 'selector-observer';
+import * as pageDetect from 'github-url-detection';
+
+import fetchDom from '../libs/fetch-dom';
+import features from '../libs/features';
+import {getRepoURL} from '../libs/utils';
+
+const getEarliestTag = cache.function(async (mergeCommit: string): Promise<string[] | undefined> => {
+	const firstAssociatedTag = await fetchDom<HTMLAnchorElement>(`/${getRepoURL()}/branch_commits/${mergeCommit}`, 'ul.branches-tag-list li:last-child a');
+	if (!firstAssociatedTag) {
+		return;
+	}
+
+	return [firstAssociatedTag.href, firstAssociatedTag.textContent!];
+}, {
+	cacheKey: ([mergeCommit]) => `earliest-tag:${getRepoURL()}/${mergeCommit}`
+});
+
+async function addEarliestTag(discussionHeader: Element) {
+	const mergeCommit = select('div.TimelineItem-body > a[href*="commit"] > code.link-gray-dark')!.textContent!;
+	const [href, textContent] = await getEarliestTag(mergeCommit) ?? [];
+
+	if (!href) {
+		return;
+	}
+
+	discussionHeader.append(
+		<span className="tooltipped tooltipped-s" aria-label={`${textContent} was the earliest tag after this PR was merged`}>
+			• <TagIcon width={14} className="d-inline-block mx-1 text-gray-light"/>
+			<span className="commit-ref css-truncate user-select-contain expandable ">
+				<a href={href}>
```

Comment:
Does this work? GitHub uses too many wraps
```suggestion
			<a href={href} className="commit-ref css-truncate user-select-contain expandable ">
```

Reply:
I was following the dom

Reply:
Before
![image](https://user-images.githubusercontent.com/16872793/81873643-2d660280-954a-11ea-9363-a5a098e91ace.png)

After
![image](https://user-images.githubusercontent.com/16872793/81873585-0e677080-954a-11ea-9140-4a5fa6b0e124.png)

Is that fine?